feeds/views.py

```
@@ -1 +1,64 @@
+            qs = qs.annotate(
+                reactions_count=Count("reactions")
+            ).order_by("-reactions_count", "-created_at")
+            return qs
+        elif sort == "old":
+            qs = qs.order_by("created_at")
+            return qs
+        return qs
+
+
+@method_decorator(cache_page(10), name='list')
+class ExploreView(viewsets.GenericViewSet, ListModelMixin):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)